src/Car.js

```diff
@@ -0,0 +1,31 @@
+import {
+  MOVE_CONDITION,
+  MOVE_COUNT,
+  START_POSITION,
+} from "./constants/setting.js";
+import { generateRandomNumber } from "./utils.js";
+
+export default class Car {
+  #name;
+  #position;
+
+  constructor(name) {
+    this.#name = name;
+    this.#position = START_POSITION;
+  }
+
+  move() {
+    const isMoving = generateRandomNumber() >= MOVE_CONDITION;
+    if (isMoving) {
+      this.#position += MOVE_COUNT;
+    }
+  }
+
+  getName() {
+    return this.#name;
+  }
+
+  getPosition() {
+    return this.#position;
+  }
```

Comment:
이럴 때는 getter를 사용해주면 어떨까요!?
```suggestion
  get name() {
    return this.#name;
  }

  get position() {
    return this.#position;
  }
```

Reply:
Airbnb JavaScript Style Guide에 따르면 getter 사용보다는 getPosition()과 같은 명시적인 메서드를 사용하라고 나와있어서 getter를 사용하지 않았습니다!

혹시 그럼에도 getter를 사용하는게 더 좋은 방식일까요?🤔
[24.2](https://github.com/airbnb/javascript?tab=readme-ov-file#accessors--no-getters-setters)

Reply:
Airbnb는 이제 좀 낡은(?) 가이드라고 생각해요 ㅋㅋ
잘쓰라고 만들어놓은 유용한 기능을 굳이 사용하지 않을 이유도 없다고 생각합니다 ㅎㅎ